### PR TITLE
Fix: Linting warning ('Avoid using id selectors')

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
Could update this - https://github.com/Financial-Times/next-graphql-api/blob/master/client/main.js#L5 - and get rid of the `id` completely

**Comment by andygout** [comment]
Node modules pulls in `graphiql.css` (from its dependency on `graphiql`); the styles therein are all dependent on the `#graphiql-container`, so we'd best keep it.

File: client/main.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import playground from './playground';
 
 window.onload = () => {
 	if (window.location.pathname === '/playground') {
-		const graphiql = document.getElementById('graphiql');
+		const graphiql = document.getElementsByClassName('graphiql')[0];
```
Comment (ironsidevsquincy) [inline client/main.js]: `querySelector` ftw

**Comment by ironsidevsquincy** [comment]
👍

**Pull request merged**